test/test_genome.py

```diff
@@ -404,6 +404,29 @@ def test_mutate_output_region(rng_seed):
         ID_NON_CODING_GENE,
     ]
 
-    assert genome._mutate_output_region(list(dna), 9, rng) is False
-    assert genome._mutate_output_region(list(dna), 10, rng) is True
-    assert genome._mutate_output_region(list(dna), 11, rng) is False
+    assert genome._mutate_output_region(list(dna), 9, rng) is True
+    assert genome._mutate_output_region(list(dna), 10, rng) is False
+    assert genome._mutate_output_region(list(dna), 11, rng) is True
+
+
+def test_correct_number_of_mutations(mutation_rate, rng_seed):
+
+    n_inputs = 2
+    n_outputs = 1
+    n_columns = 10
+    n_rows = 2
+    levels_back = 5
+    primitives = (cgp.Add, cgp.Sub, cgp.Mul, cgp.Div, cgp.ConstantFloat)
+
+    rng = np.random.RandomState(rng_seed)
+    genome = cgp.Genome(n_inputs, n_outputs, n_columns, n_rows, levels_back, primitives)
+    genome.randomize(rng)
+
+    dna_diff = 0
+    genome_new = genome.clone()
+    genome_new.mutate(mutation_rate, rng)
+    for idx_dna in range(len(genome.dna)):
+        if genome.dna[idx_dna] != genome_new.dna[idx_dna]:
+            dna_diff += 1
+
+    assert dna_diff == int(mutation_rate * len(genome.dna))
```

Comment:
```suggestion
    n_mutations_expected = int(mutation_rate * len(genome.dna))
    assert dna_diff == n_mutations_expected
```
just to make this bit a bit easier to read